jwst/tests_nightly/general/associations/test_sdp_pools.py

```diff
@@ -24,6 +25,10 @@
 # file lists for the test parametrization.
 # #############################################################
 class AssociationBase(BaseJWSTTest):
+    """Set testing environment"""
+
+    inputs_root = pytest.config.getini('inputs_root')[0]
+    results_root = pytest.config.getini('results_root')[0]
```

Comment:
I'm just highlighting this use method as I didn't notice it before @stscieisenhamer mentioned it to me.  This is an alternative way of accessing this `ini` info without having to use a fixture.  The fixture just provides the same methods as the `pytest.config` object.

Using it this way is required if one wants access to `inputs_root` during test collection.  Pytest fixtures are only available during test execution, and so cannot provide a value of `inputs_root` during test collection.  @rendinam 

So maybe we should update documentation in `ci_watson` to show that
```
def test_foo(pytestconfig):
    inputs_root = pytestconfig.getini('inputs_root')[0]
```
works for tests that need `inputs_root` during test execution.  But say a fixture or a test class setup need this information.  Then it needs to be retrieved with
```
    inputs_root = pytest.config.getini('inputs_root')[0]
```

Thanks @stscieisenhamer for discovering this little bit of undocumented `pytest` trivia.

Reply:
This is excellent. I tried something similar early on since I thought it _should_ absolutely be possible to get at pytest's config items once pytest was imported. But I don't think I attempted it with the `getini()` method. Thanks @stscieisenhamer ! I'll see that the `ci-watson` docs get updated with an example of each type of access.

Reply:
If you just do `import pytest` there is no `pytest.config` object.  Instead, it is created and is available only when `pytest` is collecting and executing tests.  Fun.

But yeah, this does open up new possibilities.